eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/ImpTest.hs

```diff
@@ -1285,3 +1286,10 @@ majorFollow pv@(ProtVer x _) = case succVersion x of
 -- | An illegal ProtVer that skips 3 minor versions
 cantFollow :: ProtVer -> ProtVer
 cantFollow (ProtVer x y) = ProtVer x (y + 3)
+
+checkProposalsEmpty :: (HasCallStack, ConwayEraImp era) => ImpTestM era ()
```

Comment:
Although slightly different (it checks the ids rather than the actions), but we have `expectNoCurrentProposals` as well.

Reply:
Oh, nice, that function seems to be doing the same thing. I might just modify them to give a more informative error message. Thanks for pointing that out!